modules/worker-utils/src/index.ts

```diff
@@ -32,6 +32,7 @@ export {createWorker} from './lib/worker-api/create-worker';
 export {getWorkerURL} from './lib/worker-api/get-worker-url';
 export {validateWorkerVersion} from './lib/worker-api/validate-worker-version';
 export {getTransferList} from './lib/worker-utils/get-transfer-list';
+export {removeNontransferableOptions} from './lib/worker-utils/remove-nontransferable-options';
```

Comment:
Why can't we just use `getTransferList`?

Reply:
I checked `getTransferList`  function and for at first sight it should help but it doesn't work as I expected.
First of all it breaks tree structure of options object.
It traverses only one level deep in recursive mode.
It replaced objects with empty array.
And finally 
![image](https://user-images.githubusercontent.com/70207219/181021349-a3dd502b-b94d-4a18-bbfe-cef5fa6c4f00.png)
It returns `ArrayBuffer`.

You can see test cases [here](https://github.com/visgl/loaders.gl/blob/master/modules/worker-utils/test/lib/worker-utils/get-transfer-list.spec.js#L16) and there are don't correspond expected result in my case.

Reply:
OK to speed this up, can you create another function in the same file, e.g. `getTransferListForWriter` and add your implementation there.

At least then I will have both of these implementations in the same place, and we can determine later if we should change the current `getTransferList()`.

It sounds strange but there may be reasons for it working the way it does so I don't want to change it now.